lib/web_ui/lib/src/engine/util.dart

```diff
@@ -649,6 +651,15 @@ int? tryViewId(Object? arguments) {
   return null;
 }
 
+int? findParentViewId(DomElement element) {
+  DomElement? current = element;
+  while (current != null && element.tagName.toLowerCase() != DomManager.flutterViewTagName) {
+    current = element.parent;
+  }
```

Comment:
I think this whole loop can be replaced by:

```dart
final DomElement? viewRoot = element.closest('${DomManager.flutterViewTagName}[${GlobalHtmlAttributes.flutterViewIdAttributeName}]');
```

That should return the closest parent to `element` (itself included) that has `tag-name[flutter-view-id]` (`tag-name[attribute-name]`).

(I'd expect the selector would be much faster than the manual parent traversal?)

Reply:
Done